opencadd/compounds/standardization/__init__.py

```diff
@@ -4,18 +4,19 @@
 """
 
 # Add imports here
-from .standardization import *
-from .assign_stereochemistry import *
-from .convert_format import *
-from .detect_inorganic import *
-from .disconnect_metals import *
-from .handle_charges import *
-from .handle_fragments import *
-from .handle_hydrogens import *
-from .normalize_molecules import *
-from .remove_salts import *
-from .sanitize_molecules import *
-from .validate_molecules import *
+from opencadd.compounds.standardization.standardization import *
+from opencadd.compounds.standardization.assign_stereochemistry import *
+from opencadd.compounds.standardization.convert_format import *
+from opencadd.compounds.standardization.detect_inorganic import *
+from opencadd.compounds.standardization.disconnect_metals import *
+from opencadd.compounds.standardization.handle_charges import *
+from opencadd.compounds.standardization.handle_fragments import *
+from opencadd.compounds.standardization.handle_hydrogens import *
+from opencadd.compounds.standardization.normalize_molecules import *
+from opencadd.compounds.standardization.remove_salts import *
+from opencadd.compounds.standardization.utils import *
+from opencadd.compounds.standardization.sanitize_molecules import *
+from opencadd.compounds.standardization.validate_molecules import *
```

Comment:
These should be relative imports. If it's not working then there's something else going on, but for sure absolute paths should not be needed.

Reply:
The absolute paths are only relevant in the pytest suite.